src/components/CalltoActionButton.js

```diff
@@ -1 +1,38 @@
-//Gets passed a prop LinkTo that leads to the desired section, exists in two different styles based on its props (https://emotion.sh/docs/styled)
\ No newline at end of file
+import styled from "@emotion/styled"
+import React from "react"
+
+const Button = styled.button`
+  border: solid 3px transparent;
+  border-radius: 100rem;
+  background-image: linear-gradient(
+      rgba(255, 255, 255, 0),
+      rgba(255, 255, 255, 0)
+    ),
+    linear-gradient(101deg, #78e4ff, #ff48fa);
+  background-origin: border-box;
+  background-clip: content-box, border-box;
+  box-shadow: 2px 1000px 1px #111 inset;
+  color: white;
+  font-family: inherit;
+  padding: ${(props) => props.size === 'small' ? '0.3rem 1rem' : '0.5rem 3rem'};
+
+  & :hover {
+    animation: pulse 3s;
+    box-shadow: 0 0 0 2em rgba(#fff, 0);
+  }
+
+  @keyframes pulse {
+    0% {
+      box-shadow: 0 0 0 0 var(--gradient);
+    }
+  }
+`
+
+
+function CalltoActionButton(props) {
+ return(
+   <Button>{props.children}</Button>
+ )
+}
```

Comment:
You can simplify this file even more because you create two components instead of one.
`styled.button` already creates a component.

See https://github.com/mschmauder-hub/seriesmaster/pull/22#discussion_r488487739

Reply:
Oh wow this is magic :D. Fixed my problem and saved code :)